src/components/repoSelector.js

```diff
@@ -0,0 +1,55 @@
+import ReactDiffViewer, { DiffMethod } from "react-diff-viewer";
+import React, { useEffect, useState } from "react";
+import { useAllOrgs, useOrgRepos } from "dcs-react-hooks";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+export default function Repository(orgName) {
+
+  const { state, action } = useOrgRepos({
+    orgName: "unfoldingWord",
+    page: 1,
+    limit: 2
```

Comment:
I would increase the limit here as well to 500 just in case the needed repo is far below the 2 limit.